opengrok-indexer/src/main/java/org/opengrok/indexer/index/Indexer.java

```diff
@@ -499,19 +504,9 @@ public static WebAddress parseWebAddress(String webAddr) {
                 cfg.setTagsEnabled(true);
             });
 
-            parser.on("-H", "--history", "=[/path/to/repository]",
-                "Get history for specific repositories (specified as",
-                "absolute path from source root), or ALL repositories",
-                "when none specified.").
-                Do( repo -> {
-                    String repository = (String) repo;
-                    if (repository.equals("")) {
-                        cfg.setHistoryEnabled(true);  // all repositories
-                    } else {
-                        repositories.add((String)repository);  // specific repository
-                    }
-                }
-            );
+            parser.on("-H", "--history", "Enable history.").Do( v -> {
```

Comment:
is the space here what idea suggests to you?
```
Do( v -> {
   ^
```

Reply:
no, I just followed the pre-existing code style in the method.

Reply:
the style doesn't seem consistent in this parser options anyway

Reply:
ok, fixed them all not to include the whitespace.

Reply:
perfect